packages/holocron-module-register-webpack-plugin/package.json

```diff
@@ -17,7 +17,7 @@
     "Nelly Kiboi <Nelly.J.Kiboi@aexp.com> (https://github.com/nellyk)",
     "Nickolas Oliver <nickolas.oliver@aexp.com> (https://github.com/PixnBits)"
   ],
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
Just a comment, but why isn't the version at the top?

Reply:
Hmmm I'm not sure..

Reply:
No need to change it right now, just thought it was odd

Reply:
We can adjust it tho